modules/ocf_backups/files/create-encrypted-backup

```diff
@@ -5,6 +5,22 @@
 # to Google Nearline, etc.
 set -o pipefail
 
+# Quiet mode is off by default (show pv output)
+QUIET=false
+
+# If -q is provided, turn on quiet mode
+while getopts "q" opt; do
+    case $opt in
+        q)
+            QUIET=true
+            ;;
+        \?)
+            # Invalid option, just exit
+            exit 1
+            ;;
+    esac
+done
```

Comment:
I don't think it's too bad really, `getopts` makes it pretty easy, but checking if the shell is interactive isn't hard, so I'll just do that instead.